documentation/release_notes.markdown

```diff
@@ -61,33 +91,56 @@ This release retires all api versions before v4 and promotes v4 to be
 the stable API. The following changes have been made since the last
 stable release of PuppetDB, which included an experimental v4 API.
 
-* Retire all previously deprecated commands ([PDB-840](https://tickets.puppetlabs.com/browse/PDB-840))
+* The version 2 and 3 of the query API are now retired, and v4 experimental is now laid out quite differently.
+
+    Full documentation outlining this change is available [in the API upgrading guide][upgrading].
+
+* We've retired some previously deprecated commands ([PDB-840](https://tickets.puppetlabs.com/browse/PDB-840))
 
     The currently supported commands and versions are now:
 
     * replace catalog, v6
     * replace facts, v4
-    * deactivate node, v2
+    * deactivate node, v3
     * store report, v5
 
-* All API commands and responses now use underscores, instead of dashes.  ([PDB-698](https://tickets.puppetlabs.com/browse/PDB-698))
+    We've insured that internally the previous version of the commands will still be processed, so you
+    don't lose the data being processed in your queue during an upgrade.
```

Comment:
Older versions of the commands on your queue will still be processed by PuppetDB so you won't lose the data from messages still being processed by PuppetDB before an upgrade.